entries/server-entry.js

```diff
@@ -68,8 +68,8 @@ export async function start({port, dir = '.'} /*: any */) {
 
 async function reload() {
   const app = await initialize();
-  reverseRegister(app, AssetsPlugin);
   reverseRegister(app, ContextPlugin);
+  app.register(app, AssetsPlugin);
```

Comment:
Registration order doesn't affect order in which plugins run. If you want to add the AssetsPlugin last, you need to modify the underlying array like reverseRegister does.

Reply:
Registration order absolutely does determine the order in which plugins are run. This code is correct as far as I can tell

Reply:
Isn't it going to get shuffled with the other ones once the depedency graph is resolved?

Reply:
It seems to work™

Reply:
Any two apps that don't depend on one another will be ordered respectively based on registration order. So this is what we want

Reply:
Ok sounds good. On a separate note, should it be `app.register(AssetsPlugin)`?

Reply:
oh yea it should be. This might be working technically because fusion would be treating `app` as a token lol. However I'm surprised this passes flow

Reply:
Yeah good catch